### PR TITLE
[doc-fix] Add missing change log entries for v0.4.0 and v0.4.1

### PR DESCRIPTION
**Pull request by LaikaN57**
Adds missing change log entries for:

- v0.4.1: https://github.com/localstack/helm-charts/pull/52
- v0.4.0: https://github.com/localstack/helm-charts/pull/48

### CODE REVIEW

**Comment by LaikaN57** [comment]
@saansh45 Please commit a fix to Nextdoor:akennedy/20220727-docfix-changelog if the line I wrote is not properly representative of the work. Give an approval when complete or if it looks fine.

@whummer Sorry for the doc errors. If you/others think the v0.4.0 description is good enough, then we can just merge this without approval from @saansh45.

**Review by saansh45 — APPROVED** [review]
👍

**Comment by whummer** [comment]
Thanks @LaikaN57 @saansh45 ! 👍 

Btw, quick note - this is largely a community-managed repo, and not super well-maintained by the LocalStack core team at this point. If you're interested in helping out with some of the maintenance (reviewing PRs, docs, adding some more features/samples), then please let me know! (just shoot me an email to waldemar@localstack.cloud). Thanks!

**Pull request merged**
